src/pages/DefaultOrgSelector/DefaultOrgSelector.jsx

```diff
@@ -134,6 +144,14 @@ function DefaultOrgSelector() {
   if (userIsLoading) return null
   if (!userIsLoading && !currentUser) return <Redirect to="/login" />
 
+  console.log('current user:', currentUser?.user?.username)
```

Comment:
remove `console.log`s

Reply:
🤦 i went back in to delete these and got distracted with fixing something else. apologies